tools/pom.xml

```diff
@@ -49,6 +49,13 @@
             <version>${log4j.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
> Do they belong there? 

I do not know for sure. I just didn't thought of a case where we won't need junit and wanted to reduce code a bit.

> where they say to children "if you need dependency A, then use version a.b" (to resolve conflicts with siblings).

ok